client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
+
+const EditReviewForm = (props) => {
+  const [editReview, setEditReview] = useState({
+    body: "",
+    score: "",
+    userId: "",
+    reviewId: "",
+    showId: ""
+  })
```

Comment:
Take another look at Nick's comment - we shouldn't need these 3 id's stored in state, we want to track in state the properties that the user would be updating (body and score)